Packs/Core/Playbooks/playbook-IOC_Alert_README.md

```diff
@@ -0,0 +1,66 @@
+IOCs provide the ability to alert on known malicious objects on endpoints across the organization. 
+
+**Analysis Actions:**
+The playbook will use serval enrichment sources to determine the IOC verdict. Additionally, will use the Analytics module to run a prevalence check for the IOC.
+
+**Response Actions**
+The playbook's first response action is a containment plan which is based on the playbook input. In that phase, the playbook will execute endpoint isolation
+
+**Investigative Actions:**
+When the playbook executes, it checks for additional abnormal activity using the Endpoint Investigation Plan playbook that can indicate the endpoint might be compromised.
+
+**Remediation Actions:**
+In case there are results found within the investigation phase, the playbook will execute remediation actions which includes containment and eradication.
+
+This phase will execute the following containment actions:
+
+* File quarantine
+* Endpoint isolation
+
+And the following eradication actions:
+
+* Manual process termination
+* Manual file deletion
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Eradication Plan
+* Enrichment for Verdict
+* Handle False Positive Alerts
+* Endpoint Investigation Plan
+* Recovery Plan
+* Containment Plan
+
+### Integrations
+This playbook does not use any integrations.
+
+### Scripts
+This playbook does not use any scripts.
+
+### Commands
+* extractIndicators
+* closeInvestigation
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| BlockIndicatorsAutomatically | Should we block suspicious/malicious indicators automatically? Specify True/False. | False | Optional |
+| ShouldCloseAutomatically | Should close the alert automatically if it's established verdict is False Positive? | True | Optional |
```

Comment:
Can you please re-run the generate-docs so the new changes you added for the inputs' description will be updated?